pkg/api/peer_req_resp.go

```diff
@@ -35,4 +35,14 @@ type PeerEditResp Peer
 type PeerGetResp Peer
 
 // PeerListResp is the response sent for a peer list request
+/*
+PeerListResp can also be filtered based on query parameters
+sent along with peer list api.
```

Comment:
`sent with peer list request`

jFYI: API is not "sent". It's an interface the client calls or interacts with.

The explanation can also be better. Something like:

```
The client can request to filter peer listing based on metadata key/value using query parameters

Example:
```